packages/scripts/src/ManualRelay.js

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+// This script allows you to manually relay a particular across deposit.
+// Example:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   ./src/ManualRelay.js --chainId 10 --depositId 100 --network mainnet_mnemonic
+// Optional env overrides:
+//   BRIDGE_ADMIN_ADDRESS
+//   BRIDGE_DEPOSIT_ADDRESS
+//   L2_END_BLOCK_NUMBER
+//
+// Note: if you want to run with GCKMS keys, the command will look like this:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   GOOGLE_APPLICATION_CREDENTIALS=/your/credentials/file.json ./src/ManualRelay.js --chainId 10 --depositId 100 \
+//   --network mainnet_gckms --keys your_gckms_key_name
+
+const argv = require("minimist")(process.argv.slice(), { string: ["depositId", "chainId"] });
+
+const { getWeb3ByChainId } = require("@uma/common");
+const { InsuredBridgeL1Client, InsuredBridgeL2Client, Logger } = require("@uma/financial-templates-lib");
+const sdk = require("@uma/sdk");
+
+const depositBoxMapping = {
+  10: "0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96",
```

Comment:
nit: why not read these from `getAddress`? You'd still need to use a constant string to fetch the contract like `getAddress("OVM_BridgeDepositBox")` but at least you don't need to repeat hardcode the address

Reply:
Oh interesting. For some reason, I thought the across addresses weren't in the main addresses files.

Reply:
Fixed.